topics/emacs/index.md

```diff
@@ -0,0 +1,13 @@
+---
+topic: emacs
+related: emacs-lisp, vim, spacemacs
+display_name: Emacs
+github_url: https://github.com/emacs-mirror/emacs
+logo: emacs.png
+short_description: An extensible, customizable, free/libre text editor — and more.
```

Comment:
Suggest using a slightly more descriptive voice here, e.g.

> Emacs is an extensible, customizable, free and libre text editor.

Reply:
Will do; honestly just lifted the description from the GNU homepage.

Reply:
Changed.